doc/shelley-genesis.md

```diff
@@ -57,7 +57,7 @@ should be created offline, kept offline and only the verification keys shared.
 
 For a demo we're going to put all the keys together in once place
 
-To start with we will set up our template directory:
+To start, with we will set up our template directory:
```

Comment:
This doesn't improve things, IMO, "with we will set up ..."  is just wrong.

I'm not convinced this needs changing, but I would go for something like "We will start with setting up ...".

Reply:
or this could be formatted as:

```suggestion
To start with, we will set up our template directory:
```